ocean_provider/validation/provider_requests.py

```diff
@@ -143,6 +176,40 @@ def rules(self):
         }
 
 
+class DecryptRequest(CustomJsonRequest):
+    def rules(self):
+        return {
+            "decrypterAddress": ["required"],
+            "chainId": ["required"],
+            "transactionId": [
+                "required_without:dataNftAddress,encryptedDocument,flags,documentHash"
+            ],
+            "dataNftAddress": [
+                "required_without:transactionId",
+                "required_with:encryptedDocument,flags,documentHash",
+            ],
+            "encryptedDocument": [
+                "required_without:transactionId",
+                "required_with:flags,documentHash",
+            ],
+            # TODO Adapt flags validation to accept "\x00"
```

Comment:
gentle reminder that you missed this TODO

Reply:
I have not researched this yet but it's definitely still on my radar.  Thanks for the reminder!